src/javascripts/main.js

```diff
@@ -1,15 +1,14 @@
 // USE WITH FIREBASE AUTH
 // import checkLoginStatus from './helpers/auth';
 import 'bootstrap'; // import bootstrap elements and js
-
 import '../styles/main.scss';
+import checkLoginStatus from './helpers/auth';
```

Comment:
You have this commented out on line 2. Delete one of them. Also, is the app starting here or checking the login status?